OIPS/OIP-0010.md

```diff
@@ -0,0 +1,117 @@
+---
+oip: 0010
+title: Anchor - Immutably Archived State Roots
+author: Martin Schenck (@schemar) <martin@ost.com>
+discussions-to: https://discuss.openst.org/t/anchor-oip-0010/54
+status: Draft
+type: Standards Track
+category (*only required for Standard Track): Mosaic
+created: 2019-01-17
+requires (*optional): none
+replaces (*optional): none
+---
+
+<!--You can leave these HTML comments in your merged OIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new OIPs. Note that an OIP number will be assigned by an editor. When opening a pull request to submit your OIP, please use an abbreviated title in the filename, `OIP-draft_title_abbrev.md`. The title should be 44 characters or less. Thanks to the Ethereum Improvement Proposal (EIP) process for the model we borrow here.-->
+# Anchor - Immutably Archived State Roots
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the OIP.-->
+By copying state roots in between blockchains, we can assert events of each
+blockchain on the respective other chain.
```

Comment:
can assert `state` (not events, they are in the log trie :P )

Reply:
Yeah yeah too much NiPoPoW lingo, will update 👍